plugin/trino-kudu/src/test/java/io/trino/plugin/kudu/TestingKuduServer.java

```diff
@@ -102,7 +102,10 @@ public TestingKuduServer(String kuduVersion)
 
     public HostAndPort getMasterAddress()
     {
-        return HostAndPort.fromParts(master.getContainerIpAddress(), master.getMappedPort(KUDU_MASTER_PORT));
+        // Do not use master.getContainerIpAddress(), it returns "localhost" which the kudu client resolves to:
```

Comment:
Make link clickable in commit message by adding a `https://` in front.

Reply:
I had that originally and the link was large than the commit max line length 😅

Reply:
You can refer to them like `apache/kudu@<hash>` too I believe. Or abbreviate the commit hash until it fits (any length works as long as the hash is not ambiguous).